.github/workflows/validate-pr.yml

```diff
@@ -1,21 +1,24 @@
+# SPDX-License-Identifier: Apache-2.0
+# SPDX-FileCopyrightText: Copyright 2024 The SPDX Contributors
+
 name: validate PR
 
 on:
-  pull-request
+  pull_request:
```

Comment:
I don't think the final `:` is actually needed, if there is nothing underneath in inner levels.